### DISCUSSION TITLE
sub menu item

### DISCUSSION THREAD

**Discussion by nicboul** [progrium/darwinkit#27] [Q&A] [answered]
Hello, how do I add a sub menu item ? thanks ! ( I'm playing with the examples/pomodoro example )

**Comment by nicboul** [comment, marked as answer]
Ok I found the answer, you first need to create a menuItem, then create a menu2, and set that menu2 as submenu of the menuItem

	aMenuItem.SetSubmenu(aMenu)

then: aMenu.AddItem(anotherItem)

**Reply by progrium** [reply]
exactly, thanks for sharing!
